core/unix/os_exports.h

```diff
@@ -351,7 +351,7 @@ extern app_pc vsyscall_sysenter_displaced_pc;
 #define VSYSCALL_PAGE_MAPS_NAME "[vdso]"
 
 bool
-is_thread_create_syscall(dcontext_t *dcontext);
+is_thread_create_syscall(dcontext_t *dcontext _IF_LINUX(void *maybe_clone_args));
```

Comment:
This is no longer exported so it should not be in os_exports.h anymore.

Reply:
Done.